snapcraft/internal/meta/_snap_packaging.py

```diff
@@ -527,7 +576,15 @@ def _write_wrap_exe(self, wrapexec, wrappath, shebang=None, args=None, cwd=None)
             if assembled_env:
                 print("{}".format(assembled_env), file=f)
                 print(
-                    "export LD_LIBRARY_PATH=$SNAP_LIBRARY_PATH:$LD_LIBRARY_PATH", file=f
+                    'export LD_LIBRARY_PATH="$SNAP_LIBRARY_PATH${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}"',
+                    file=f,
+                )
+                print(
+                    'echo $LD_LIBRARY_PATH | grep -qE "::|^:|:$" && '
```

Comment:
This doesn't make sense to me.  Snapcraft should be erroring instead of spamming the user.

Snapcraft is far more likely to break applications now by writing to stdout. (Think any app that has parsed output).

Reply:
removed